odoo/tools/convert.py

```diff
@@ -533,6 +533,7 @@ def _tag_record(self, rec):
                 raise Exception("Cannot update missing record %r" % xid)
 
         res = {}
+        delay_list = []
```

Comment:
I would call that `sub_records`, because that's what it is.
The fact that it is processed after the main record is an implementation detail.